plugins/cd/__init__.py

```diff
@@ -25,27 +25,32 @@
 # from your version.
 
 
+from six import range
```

Comment:
> Is there recommended way to cover changes in plugin dirs ?

I don't understand what you mean. Can you please explain more detailled or give an example?